src/setting_templates/Settings_box1.csv

```diff
@@ -4,3 +4,6 @@ BonsaiOsc1,4002
 BonsaiOsc2,4003
 BonsaiOsc3,4004
 BonsaiOsc4,4005
+HighSpeedCamera,0
+SideCamera,23022724
```

Comment:
The SideCamera and BottomCamera values should default to 0 (1) for clarity, and (2) the values will differ in different ephys rigs, right? So it doesnt make to default to a specific rig

Reply:
I changed it to 12345678 to differentiate between the Boolean and the serial number.